dev-tools/src/lambdas/addFirehoseRecord/handler.ts

```diff
@@ -9,11 +9,6 @@ export const handler = async (
   context: Context
 ) => {
   initialiseLogger(context)
-  logger.info(
```

Comment:
For these logs that show the function parameter... is this not useful information? In my mind this is how you would identify which group of logs is relevant to your invocation. E.g. zendesk id, etc.

Reply:
I've just added a log here of the event_id so it's possible to track a request going through. Do you think this is enough info or should we log more? I realise this is a dev tool and won't be deployed in prod or anything, but was just thinking it would be good to keep to a consistent practice of not logging the entire parameter when it comes in. What do you think?

Reply:
eventId's seem sensible

Reply:
Yep agree.